src/types/onyx/Policy.ts

```diff
@@ -57,31 +57,34 @@ type Attributes = {
 };
 
 /** Policy custom unit */
-type CustomUnit = OnyxCommon.OnyxValueWithOfflineFeedback<{
-    /** Custom unit name */
-    name: string;
+type CustomUnit = OnyxCommon.OnyxValueWithOfflineFeedback<
```

Comment:
Was this change intended?

Reply:
Yes, we added `keyof Attributes` at the end so it'll have this format